### PR TITLE
Release/2.7.0

### PR DESCRIPTION
**Pull request by jholloc**
Major changes:

- Addition of Capn Protocol as a serialisation technology available in the UDA plugins - this is made available using the CMake ENABLE_CAPNP flag, which defaults to OFF
- Addition of new C++ client. Currently this client is working but is not used by any of the wrappers. After further testing this will become the default client in UDA 3.0
- Improving GitHub CI integration

### CODE REVIEW

File: LICENCE.txt [hunk 1]
```diff
@@ -1,20 +1,209 @@
...
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright (c) 2017-2022, UKAEA/Euratom Fusion Association
+   Culham Laboratory, Abingdon, Oxon, OX14 3DB, UK
+
+   Copyright (c) 2017-2022, CEA, IRFM
+   F-13108 Saint-Paul-lez-Durance, France
+
+   ITER Organization
```
Comment (SimonPinches) [inline LICENCE.txt:196]: I would suggest to add the copyright years in the same way as for the other contributers.
```suggestion
   Copyright (c) 2017-2022, ITER Organization
```

File: scripts/iter-ci-setup-env.sh [hunk 2]
```diff
@@ -3,80 +3,80 @@
...
+    # Stage 0 : load modules
 
-	# Set up environment for compilation
-	. /usr/share/Modules/init/sh
-	module use /work/imas/etc/modulefiles
-	module use /work/imas/etc/modules/all
-	module purge
+    # Set up environment for compilation
+    . /usr/share/Modules/init/sh
+    module use /work/imas/etc/modulefiles
```
Comment (SimonPinches) [inline scripts/iter-ci-setup-env.sh:67]: Not necessary:
```suggestion
```

**Review by SimonPinches — COMMENTED** [review]
A few minor suggestions.

**Comment by olivhoenen** [comment]
@jholloc maybe worth adding Lana Abadie to this PR w.r.t to CODAC's use of UDA.

**Review by olivhoenen — APPROVED** [review]

**Review by SimonPinches — APPROVED** [review]

**Pull request merged**